src/util/metadata_handler.ts

```diff
@@ -0,0 +1,21 @@
+import { ValueID, ValueMetadata, ZWaveNode } from "zwave-js";
+
+export const getTransformedValueMetadata = (
```

Comment:
Should we move these functions to `state.ts`, as that is responsible for serializing the state.